src/validators/comment-validator.js

```diff
@@ -0,0 +1,27 @@
+const  {check}  =  require ( 'express-validator' );
+const {validationAccountId, validationComment} = require('../validators/discussion-validator');
+const {validationId} = require('../validators/account-validator');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
const {validationId} = require("../validators/account-validator");
```